### DISCUSSION TITLE
Think & TODO

### DISCUSSION THREAD

**Discussion by G3G4X5X6** [Yong-An-Dang/nuclei-plus#20] [Ideas]
欢迎大家在下面提出自己的想法和期待实现的功能！

**Comment by Dragon-Tab** [comment]
期待Searching功能添加一个根据ip、端口去重的功能，例如以下三条去重后变为一条
![1705925121601](https://github.com/Yong-An-Dang/nuclei-plus/assets/53103912/3791b40c-aef0-4a56-8629-7a73490aa7cc)
期待新建模板功能，加载默认或用户可自定义的模板为新模板，或根据http请求包直接生成模板
![1705925398993](https://github.com/Yong-An-Dang/nuclei-plus/assets/53103912/09569df7-5b34-40f1-8a4f-0cd1dadfce38)
